truffle/LoadContract.html

```diff
@@ -53,31 +56,38 @@
 deactivate L
 end
 
-S ->> L: setCarrier()
+S ->>V:registerVault(vaultId, vaultUri, vaultHash)
+activate V
+note over V: emit VaultRegistered
+deactivate V
+
+S ->> L: setCarrier(vaultId, Carrier)
```

Comment:
I think we should update mermaid to show the flow for createNewShipment2, and we don't need to keep the mermaid for createNewShipment. Be mindful of the event emission order.